src/System.Private.CoreLib/shared/System/IO/Path.Unix.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
GH won't let me comment but since we haven't annotated Environment.GetEnvironmentVariable which can return null, we will have to revisit this file when we do so, if you want to fix it now instead, this variable should be nullable: https://github.com/dotnet/coreclr/pull/23732/files#diff-553ad0014ae47a55043328c809d13346R81

Reply:
Fixed.